src/btree/bt_vrfy.c

```diff
@@ -699,16 +705,22 @@ __verify_tree(
             ++vs->depth;
             ret = __wt_page_in(session, child_ref, 0);
 
-            /*
-             * If configured, continue traversing through the pages of the tree even after
-             * encountering errors reading in the page.
-             */
-            if (vs->read_corrupt && ret != 0) {
+            if (ret != 0) {
+                if (vs->dump_address)
+                    WT_TRET(__wt_msg(session,
+                      "%s Read failure while accessing a page from the row internal page (ret = "
+                      "%d).",
+                      __verify_addr_string(session, ref, vs->tmp1), ret));
```

Comment:
Likewise, `ref` here should be `child_ref`.

Reply:
Oh yes, thank you Keith! Fixed, https://github.com/wiredtiger/wiredtiger/pull/10095/commits/133dbfe536f2b11b1c10d5e4c8733ddaad4dd3cb